app/v3/connectors/ListPeriodSummariesConnector.scala

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Do we need to create a v2 `ListPeriodSummariesConnector` as well? We might need to keep v2 up to date because it's still in beta. Currently the v2 list endpoint falls back to the v1 implementation, which we can't update because v1 is deprecated.